File: examples/src/main/scale/kafka/examples/CreateTopicDemo.scala
package kafka

import org.apache.kafka.clients.admin.AdminClient

import java.util
import java.util.Collections

object CreateTopicDemo {
  def main(args: Array[String]): Unit = {
    val props = new util.HashMap[String, Object]()
    props.put("bootstrap.servers", "127.0.0.1:9093")
    val adminClient = {
      AdminClient.create(props)
    }
    adminClient.createTopics(Collections.singleton("test-topic")).all().get()
  }
}

File: examples/src/main/java/kafka/examples/CreateTopicDemo.java
package kafka.examples;
import org.apache.kafka.clients.admin.AdminClient;
import org.apache.kafka.clients.admin.NewTopic;

import java.util.Collections;
import java.util.HashMap;
import java.util.Map;
import java.util.concurrent.ExecutionException;
public class CreateTopicDemo {
		public static void main(String[] args) throws ExecutionException, InterruptedException {
				Map<String, Object> props = new HashMap<>();
				props.put("bootstrap.servers", "127.0.0.1:9093");
				AdminClient adminClient = AdminClient.create(props);
				adminClient.createTopics(Collections.singletonList(new NewTopic("test-topic", 2,
					(short) -1))).all().get();
		}
}
